unified-codes/apps/web/src/components/templates/Explorer.tsx

```diff
@@ -107,6 +107,10 @@ export const ExplorerComponent: Explorer = ({
 
   return (
     <EntityBrowser
+      orderDesc={variables.orderDesc}
+      orderBy={variables.orderBy}
+      page={variables.page}
+      rowsPerPage={variables.rowsPerPage}
```

Comment:
Ignore this for now, haven't refactored the web app to use the new component API yet!